lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1714,15 +1714,19 @@ def init_api_client
     def api_client
       if @use_grpc
         ssl_creds = GRPC::Core::ChannelCredentials.new
-        authentication = Google::Auth.get_application_default
-        creds = GRPC::Core::CallCredentials.new(authentication.updater_proc)
+        authentication = Google::Auth.get_application_default(LOGGING_SCOPE)
+        updater_proc = lambda do |a_hash|
+          authentication.apply(a_hash, use_configured_scope: true)
+        end
+        creds = GRPC::Core::CallCredentials.new(updater_proc)
```

Comment:
You could inline the `lambda` expression directly into the `GRPC::Core::CallCredentials.new` call, i.e.,
```ruby
        creds = GRPC::Core::CallCredentials.new(lambda do |a_hash|
          authentication.apply(a_hash, use_configured_scope: true)
        end)

Reply:
Done.